Classes/KIFTestActor.m

```diff
@@ -179,7 +180,13 @@ - (void)waitForTimeInterval:(NSTimeInterval)timeInterval
 }
 
 -(void)waitForAnimationsToFinish {
-    NSTimeInterval maximumWaitingTimeInterval = 3;
+    NSTimeInterval maximumWaitingTimeInterval = self.animationWaitingTimeout;
+    if(maximumWaitingTimeInterval <= 0) {
+        return;
+    }
+    
+    // Wait for the view to stabilize and give them a chance to start animations before we wait for them.
+    [self waitForTimeInterval:0.5f];
```

Comment:
The original 0.5 second wait was to allow animations to complete, so it was really a poor man's `waitForAnimationsToFinish`. Now that we have a real function to do this, we do not need the callers in KIFUITestActor.m to manually do the wait anymore. Furthermore, if a client sets the `maximumWaitingTimeInterval` to 0, this allows them to skip the whole stabilization delay, which is a good thing. On the other hand, you did find a bug: if the `maximumWaitingTimeInterval` is less than 0.5 seconds, then we should call waitForTimeInterval using the maximum and then get out. I filed issue #550 to cover this.